sw/lib/ocaml/compat.ml

```diff
@@ -75,3 +69,39 @@ let bytes_set = fun s n c->
 
 let bytes_iter = fun f s->
   BYTES.iter f s
+
+IFDEF OCAML_V404 THEN
```

Comment:
Since uppercase/lowercase/capitalize functions from Bytes and String modules seems to have the same effect, I think we can make something simpler like:
```
IFDEF OCAML_V404 THEN
let lowercase_ascii = BYTES.lowercase_ascii
let uppercase_ascii = BYTES.uppercase_ascii
let capitalize_ascii = BYTES.capitalize_ascii
ELSE
let lowercase_ascii = BYTES.lowercase
let uppercase_ascii = BYTES.uppercase
let capitalize_ascii = BYTES.capitalize
END
```
This way the functions from `Compat` will match the new standards and `BYTES` will pick the correct module.